### PR TITLE
added common FAQs

### PR DESCRIPTION
**Pull request by Nixon-Samuel**

### CODE REVIEW

File: components/Content.tsx [hunk 1]
```diff
@@ -27,9 +27,9 @@ const data = {
     network: Network,
     token: Token,
     tokenEndpoint: TokenEndpoint,
-    securityTokens1: SecurityTokens1,
-    securityTokens2: SecurityTokens2,
-    firewallAndPorts: FirewallAndPorts,
+    securityAndTokens1: SecurityTokens1,
```
Comment (KaustubhKumar05) [inline components/Content.tsx:30]: Can you change these back to securityTokensx @Nixon-Samuel?
The component name needs to be the same as the file name - the 'and' is extra here, and the Algolia script won't be able to replace the content

Comment (Nixon-Samuel) [reply components/Content.tsx]: done

**Review by KaustubhKumar05 — APPROVED** [review]

**Comment by ygit** [comment]
Nice 🙌

File: components/SearchModal.tsx [hunk 2]
```diff
@@ -53,7 +53,7 @@ const Result = ({ searchResult }) => {
                 }}>
                 {path.map((text, id) =>
                     id === path.length - 1 ? (
-                        text
+                        text.split('#')[0]
```
Comment (KaustubhKumar05) [inline components/SearchModal.tsx:56]: Does not show FAQ ID in the path

File: lib/withTableofContents.ts [hunk 3]
```diff
@@ -10,7 +10,7 @@ const withTableofContents = (toc?: any[]) => () => (tree) => {
     // @ts-ignore
     for (let i = 0; i < tree.children.length; i++) {
       const node = tree.children[i]
-      if (node.type === 'heading' && [2, 3].includes(node.depth)) {
+      if (node.type === 'heading' && [2, 3, 4].includes(node.depth)) {
```
Comment (KaustubhKumar05) [inline lib/withTableofContents.ts:13]: h4s will also be assigned IDs

File: lib/algolia/algoliaUtils.js [hunk 4]
```diff
@@ -134,7 +134,7 @@ exports.createRecords = function createRecords(
                         basePath,
                         contentAlias,
                         getSectionContent,
-                        content.includes('faq.mdx')
+                        false
```
Comment (KaustubhKumar05) [inline lib/algolia/algoliaUtils.js:137]: FAQs are directly being sourced from a google sheet

**Review by hdz-666 — APPROVED** [review]

**Pull request merged**